cmd/operator/main.go

```diff
@@ -114,6 +116,8 @@ func init() {
 	flag.IntVar(&maxPodRestartsPerMinute, "max-pod-restarts-per-minute", defaultMaxPodRestartsPerMinute, "Maximum number of pods in an invalid state that can be restarted per minute")
 	flag.BoolVar(&disableLeaderElection, "disable-leader-election", false, "Disable leader election for operator")
 
+	flag.StringVar(&watchNamespace, "watch-namespace", "", "Namespace to watch K8s resources")
```

Comment:
`Namespace to watch K8s resources in` or `Namespace to watch for K8s resources`

Reply:
Namespace to watch for Dapr annotated pod deployments

Reply:
Rephrased